client/src/components/ProfileTabs/ProfileTabs.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Tabs } from 'antd';
+import './ProfileTabs.css';
+
+const ProfileTabs = () => (
+  <div>
+    <Tabs defaultActiveKey="1" className="Tabs__section" centered>
+      <Tabs.TabPane tab="Upcoming interviews" key="1" defaultActiveKey className="Tab__header">
+        Contents of Tab Pane 1
+      </Tabs.TabPane>
+      <Tabs.TabPane tab="Reviews" key="2">
+        Content of Tab Pane 2
+      </Tabs.TabPane>
+      <Tabs.TabPane tab="Settings" key="3">
```

Comment:
Maybe it's better to arrange the tabs `Upcoming interviews` then `Interviews history` then `Reviews` then `Settings.